src/release_artifacts_resources/ios/cdk/cdk/credential_rotation/lambda_functions/src/source_data_generator/aws_credential_rotator.py

```diff
@@ -0,0 +1,96 @@
+import boto3
```

Comment:
1. This is now not really an `aws_credential_rotator` so much as an `aws_session_credential_source` or similar, right?
2. Doc comment:
```suggestion
"""Creates static credentials, uses them to adopt the specified role and get temporary session credentials, and deletes the static credentials.

import boto3
```